frame/bridge/ethereum/issuing/src/lib.rs

```diff
@@ -355,8 +355,7 @@ impl<T: Config> Pallet<T> {
 		let factory_address = MappingFactoryAddress::<T>::get();
 		let bytes = mtf::encode_mapping_token(backing, source)
 			.map_err(|_| Error::<T>::InvalidIssuingAccount)?;
-		let mapped_address = dvm_ethereum::Pallet::<T>::do_call(factory_address, bytes)
-			.map_err(|e| -> &'static str { e.into() })?;
+		let mapped_address = dvm_ethereum::Pallet::<T>::raw_call(factory_address, bytes)?;
```

Comment:
`raw_call` is readonly, will it write some thing here?

Reply:
No, just a read-only call to the smart contract and catch the return value.